### PR TITLE
Autofill: loose end tidy up

### PR DESCRIPTION
**Pull request by CDRussell**
Task/Issue URL: https://app.asana.com/0/414730916066338/1202554588881474/f

### Steps to test this PR
Code review only

### CODE REVIEW

File: app/src/main/res/layout/content_settings_privacy.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
         style="@style/SettingsItemClickable"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        android:text="[l10n] Autofill"
+        android:text="@string/autofillSettingsDisplayName"
```
Comment (karlenDimla) [inline app/src/main/res/layout/content_settings_privacy.xml:56]: Do we have a guideline when adding strings? Should it be camel case or _separated? I've seen both patterns on the project.

File: modules/autofill/autofill-api/build.gradle [hunk 2]
```diff
@@ -40,9 +40,9 @@ dependencies {
     implementation AndroidX.lifecycle.runtimeKtx
     implementation AndroidX.lifecycle.viewModelKtx
     implementation Google.android.material
-    implementation 'androidx.appcompat:appcompat:1.4.1'
-    implementation 'com.google.android.material:material:1.6.0'
-    implementation 'androidx.constraintlayout:constraintlayout:2.1.3'
+    implementation 'androidx.appcompat:appcompat:_'
```
Comment (karlenDimla) [inline modules/autofill/autofill-api/build.gradle:43]: We could use: implementation AndroidX.appCompat

File: modules/autofill/autofill-api/build.gradle [hunk 3]
```diff
@@ -40,9 +40,9 @@ dependencies {
...
+    implementation 'com.google.android.material:material:_'
```
Comment (karlenDimla) [inline modules/autofill/autofill-api/build.gradle:44]: We could use: implementation Google.android.material

File: modules/autofill/autofill-api/build.gradle [hunk 4]
```diff
@@ -40,9 +40,9 @@ dependencies {
...
+    implementation 'androidx.constraintlayout:constraintlayout:_'
```
Comment (karlenDimla) [inline modules/autofill/autofill-api/build.gradle:45]: We could use: implementation AndroidX.constraintLayout

File: app/src/main/res/layout/content_settings_privacy.xml (hunk 1 above)
Comment (CDRussell) [reply app/src/main/res/layout/content_settings_privacy.xml:56]: Yeah both are used in the codebase. Probably more camel case than anything else.

File: modules/autofill/autofill-store/build.gradle [hunk 5]
```diff
@@ -41,27 +13,45 @@ dependencies {
...
+    implementation AndroidX.appCompat
     implementation KotlinX.coroutines.core
     implementation Google.android.material
     implementation project(path: ':secure-storage-api')
+
+    implementation Google.dagger
+    implementation JakeWharton.timber
+
+    // Testing dependencies
+    testImplementation AndroidX.work.testing
```
Comment (karlenDimla) [inline modules/autofill/autofill-store/build.gradle:26]: Why are we adding all this new modules when we haven't added any thing new in the tests in this module?

Comment (CDRussell) [reply modules/autofill/autofill-store/build.gradle]: I was having problems with the unit tests failing on this module, and re-structuring the build.gradle was part of my attempt at fixing it. I believe it was likely a local cacheing issue, so reverted some of these changes. Kept a few others that seemed like a good improvement anyways.

**Pull request merged**